source/jormungandr/jormungandr/interfaces/v1/serializer/base.py

```diff
@@ -169,8 +169,9 @@ def to_value(self, obj):
 
 
 class GenericSerializer(PbNestedSerializer):
-    id = jsonschema.Field(schema_type=str, attr='uri', description='Identifier of the object')
-    name = jsonschema.Field(schema_type=str, description='Name of the object')
+    id = jsonschema.Field(schema_type=str, required=True, attr='uri',
```

Comment:
Should we use `required=True` or `display_none=True`?

Reply:
Since serpy sets `required` to false if `display_none` is absent of false... `display_none` is our champion :tada: